plugin-server/src/worker/ingestion/team-manager.ts

```diff
@@ -334,12 +341,20 @@ DO UPDATE SET property_type=$5, property_type_format=$6 WHERE posthog_propertyde
                 'fetchPropertyDefinitions'
             )
 
-            this.propertyDefinitionsCache.set(
-                teamId,
-                eventProperties.rows.reduce((teamCache, row) => {
-                    teamCache[row.name] = row.property_type ?? NULL_IN_DATABASE
-                    return teamCache
-                }, new Map())
+            const teamPropertyDefinitionsCache = new LRU<string, string | symbol>({
+                max: this.lruCacheSize * 10, // keep in memory the last 100k property definitions we have seen
```

Comment:
Where do we get this magic constant `10`? I'd split up the original ENV and make this explicit.

Reply:
Was going off 200k above and trying not to store them all. Have knocked it back to the `lruCacheSize`. Let's release it and use the statsd logging to see if it needs tuning